docs/index.md

```diff
@@ -0,0 +1,23 @@
+---
+
+---
+
+# CLDR Docs
+
+CLDR main page: [https://unicode.org/cldr](https://unicode.org/cldr)
+
+## TR35
+
+- [Unicode Technical Standard #35 UNICODE LOCALE DATA MARKUP LANGUAGE (LDML)](ldml/tr35.html)
+
+## RFC
+
+- draft-davis-t-langtag-ext: [html](rfc/draft-davis-t-langtag-ext.html) [txt](rfc/draft-davis-t-langtag-ext.txt)
+
+- draft-davis-u-langtag-ext-00: [html](rfc/draft-davis-u-langtag-ext-00.html) [txt](rfc/draft-davis-u-langtag-ext-00.txt)
+
+##### Copyright
+
+Copyright &copy; 1991-2019 Unicode, Inc.
```

Comment:
Could we generate and use the current year instead of "1991-2019"?

Maybe something like {{ site.time | date: '%Y' }}

https://michaelsoolee.com/jekyll-copyright/

https://www.alanwsmith.com/jekyll-liquid-date-formatting-examples

Reply:
Tom has a question pending...

Reply:
> Could we generate and use the current year instead of "1991-2019"?
> 
> 
> 
> Maybe something like {{ site.time | date: '%Y' }}
> 
> 
> 
> https://michaelsoolee.com/jekyll-copyright/
> 
> 
> 
> https://www.alanwsmith.com/jekyll-liquid-date-formatting-examples

Could this be a follow on?

Reply:
fine with me, and optional as far as I'm concerned -- I was just wondering